test/e2e/run-all.sh

```diff
@@ -1,12 +1,33 @@
 #!/usr/bin/env bash
 
+set -x
 set -e
 set -u
 set -o pipefail
 
 export PATH="$PATH:./node_modules/.bin"
+export GANACHE_ARGS='--quiet --blockTime 2'
```

Comment:
The `--quiet` flag doesn't seem to be doing the trick either - the logs are still full of messages from `ganache`

Reply:
[<img width="1016" alt="" src="https://user-images.githubusercontent.com/1623628/60903156-64dd9180-a24b-11e9-900c-f2f889c106dc.png">](https://github.com/trufflesuite/ganache-cli/commit/e89d4325b94dd74e7f71a1adbc44d982c1b244d1)

Simply enough, the flag is newer than our version of the CLI. I'll update our CLI version.